src/carousel.js

```diff
@@ -1,6 +1,7 @@
 import Vue from "vue";
 import VueResource from "vue-resource";
 import Carousel from "./components/Carousel.vue";
+import styles from './carousel.css';
```

Comment:
Good question. I'm not an expert in webpack (in fact I mostly refuse to use it). From a dev perspective the source files are now separate (although long-term I'd like it to be SASS / SCSS and output a minified css file.) For now this seems a nice compromise.